checker/raw_result.go

```diff
@@ -143,8 +169,31 @@ type Review struct {
 
 // User represent a user.
 type User struct {
-	Login string
-}
+	RepoAssociation *RepoAssociation
```

Comment:
For consistency with `clients/` package and GitHub data model, this should be included in `Comment` struct as `authorAssociation`.